pim-tool-front/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "elca",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --open",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^16.1.0",
```

Comment:
[S] Always use the exact version, avoid using `~` and `^` because they will automatically upgrade the dependencies if there is new version. This will cause instability during development and testing, especially if the developer of the library accidentally introduce a breaking bug in newer version.